src/main.rs

```diff
@@ -1417,8 +1441,8 @@ mod tests {
         assert!(Decoder::compile(&f, &Next::Empty).is_err());
     }
 
-    // This test takes over a minute to run
-    //#[test]
+    #[test]
+    #[ignore] // This test takes over a minute to run
```

Comment:
Very minor (feel free to ignore):

```suggestion
    #[ignore = "this test takes over a minute to run"]
```

Reply:
Oooh I didn't know about that!

Reply:
Ooh nice metadata!